proto/sedachain/wasm_storage/query.proto

```diff
@@ -0,0 +1,65 @@
+syntax = "proto3";
+package sedachain.wasm_storage;
+
+import "gogoproto/gogo.proto";
+import "google/api/annotations.proto";
+import "sedachain/wasm_storage/wasm_storage.proto";
+
+option go_package = "github.com/sedaprotocol/seda-chain/x/wasm-storage/types";
+
+// Query defines the gRPC querier service.
+service Query {
+  // DataRequestWasm returns Data Request Wasm given its hash.
+  rpc DataRequestWasm(QueryDataRequestWasmRequest)
+      returns (QueryDataRequestWasmResponse) {
+    option (google.api.http).get =
+        "/seda-chain/wasm-storage/data_request_wasm/{hash}";
+  }
+
+  rpc DataRequestWasms(QueryDataRequestWasmsRequest)
+      returns (QueryDataRequestWasmsResponse) {
+    option (google.api.http).get =
+        "/seda-chain/wasm-storage/data_request_wasms";
+  }
+
+  // OverlayWasm returns Overlay Wasm given its hash.
+  rpc OverlayWasm(QueryOverlayWasmRequest) returns (QueryOverlayWasmResponse) {
+    option (google.api.http).get =
+        "/seda-chain/wasm-storage/overlay_wasm/{hash}";
+  }
+
+  rpc OverlayWasms(QueryOverlayWasmsRequest)
+      returns (QueryOverlayWasmsResponse) {
+    option (google.api.http).get = "/seda-chain/wasm-storage/overlay_wasms";
+  }
+}
+
+message QueryDataRequestWasmRequest {
+  string hash = 1 [ (gogoproto.moretags) = "yaml:\"hash\"" ];
```

Comment:
Possibly a dumb question:

What's the `yaml` specifier for here in these messages? Isn't `json` more standard for `rpc` things?

Reply:
Not dumb at all.. probably for query results? Copy-pasting went too far there. I removed them.